Co-Scientist addition to billing page (#1371)
seqeralabs/docs: platform-cloud/docs/administration/credit-management.md

---
title: "Billing and credit management"
description: "Manage Seqera credits across Seqera Compute and Co-Scientist."
date created: "2025-10-20"
last updated: "2026-04-27"
tags: [seqera compute, compute environment, co-scientist, billing]
---

> Seqera Compute environments and Co-Scientist both consume credits from a shared credit pool. Compute credits are deducted in real time at task completion. Co-Scientist usage is included up to a per-plan monthly allowance, with credits charged only for usage above that allowance.

Reviewer: ```suggestion
Seqera Compute environments and Co-Scientist share a credit pool at the organization level. Each user also receives a monthly Co-Scientist allowance based on their plan. Once a user exhausts their allowance, further Co-Scientist usage draws from the shared pool. Compute credits are deducted from the shared pool in real time at task completion.
```
I found the wording ambiguous. This is my understanding.

| **Storage**          | 0.025          | GB-month     | Actual usage                | Daily reconciliation  | S3 storage costs at AWS rates, varies by region                                                                               |
| **Network transfer** | Varies by region | GB         | Actual data transfer        | Daily reconciliation  | Data egress charges at AWS rates. Network charges reflect in billing reports after 48 hours.                                  |

:::info
CPU and memory are billed based on **requested** resources in your pipeline configuration, not actual usage. Storage and network costs are billed based on actual consumption.
:::

### Co-Scientist usage

Co-Scientist follows a tiered billing model:

- **Included usage**: Each plan includes a monthly allowance of Co-Scientist usage. Usage within this allowance is not charged. The included allowance resets monthly.
- **Credit overage**: Usage above the included allowance is billed against your organization's credit balance, deducted per AI inference call.

> The **Credit usage** dialog in Co-Scientist shows your usage on both tiers — included usage with a monthly progress bar, and remaining org credits with the credit expiration date. When included usage is exhausted, requests automatically continue to draw from credits if available.

Reviewer: ```suggestion
The **Credit usage** dialog in Co-Scientist shows your usage on both tiers, including included usage with a monthly progress bar and remaining org credits with the credit expiration date. When included usage is exhausted, requests automatically draw from available credits.
```

| PREPARE_GENOME:GUNZIP_ADDITIONAL_FASTA (gfp.fa.gz)                            | 3 m 28 s | 1    | 6.0 GB  |
| PREPARE_GENOME:UNTAR_SALMON_INDEX (salmon.tar.gz)                             | 3 m 28 s | 1    | 6.0 GB  |
| PREPARE_GENOME:GUNZIP_GTF (genes_with_empty_tid.gtf.gz)                       | 3 m 28 s | 1    | 6.0 GB  |
| RNASEQ:FASTQ_QC_TRIM_FILTER_SETSTRANDEDNESS:CAT_FASTQ (WT_REP1)               | 3 m 29 s | 1    | 6.0 GB  |
| RNASEQ:FASTQ_QC_TRIM_FILTER_SETSTRANDEDNESS:FQ_LINT (WT_REP2)                 | 3 m 29 s | 2    | 12.0 GB |
| RNASEQ:FASTQ_QC_TRIM_FILTER_SETSTRANDEDNESS:FQ_LINT (RAP1_UNINDUCED_REP1)     | 3 m 29 s | 2    | 12.0 GB |
| RNASEQ:FASTQ_QC_TRIM_FILTER_SETSTRANDEDNESS:CAT_FASTQ (RAP1_UNINDUCED_REP2)   | 3 m 29 s | 1    | 6.0 GB  |
| RNASEQ:FASTQ_QC_TRIM_FILTER_SETSTRANDEDNESS:FQ_LINT (RAP1_IAA_30M_REP1)       | 3 m 29 s | 2    | 12.0 GB |
| PREPARE_GENOME:GTF_FILTER (genome.fasta)                                      | 47 s     | 1    | 6.0 GB  |
| FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (WT_REP2)                             | 46 s     | 4    | 15.0 GB |

:::info

> The **Tasks** tab of the [run details](https://docs.seqera.io/platform-cloud/monitoring/run-details) page lists the processes and tasks executed during the run. Select a task from the list to view the task's details, including metrics for **Execution time** and **Resources requested**.

Author: ```suggestion
The **Tasks** tab of the [run details](../monitoring/run-details) page lists the processes and tasks executed during the run. Select a task from the list to view the task's details, including metrics for **Execution time** and **Resources requested**.
```

> The **Tasks** tab of the [run details](https://docs.seqera.io/platform-cloud/monitoring/run-details) page lists the processes and tasks executed during the run. Select a task from the list to view the task's details, including metrics for **Execution time** and **Resources requested**.

Reviewer: Don't need the full link here?

- Aggregated costs (not individual task breakdowns)

### Request additional credits

To request more credits:

1. Select **Request more credits** in the organization or workspace settings **Credits** view.
2. Complete the form with your contact, organization, and credit request details.
3. Credits are typically allocated within one business day.

:::info
[**Request credits**](https://seqera.io/platform/compute/request-credits/) online or contact your Seqera account manager for assistance.
:::

### Credit limits and service suspension

When your organization or workspace credit balance is exhausted, all credit-billable services are affected:

1. **Running pipelines paused**: All active pipeline runs and Studio sessions are automatically suspended.
2. **Seqera Compute buckets locked**: Data can no longer be browsed or downloaded from Data Explorer.
3. **New launches blocked**: No new pipeline runs or Studios can be started using Seqera Compute environments.
4. **Co-Scientist requests blocked**: Once both your included monthly AI usage and your org credit balance are exhausted, new requests are blocked until credits are added.

> 5. **Resume runs manually**: After purchasing additional credits, manually [resume](https://docs.seqera.io/platform-cloud/launch/cache-resume) paused pipelines.

Reviewer: Same repo, could be relative?

> 5. **Resume runs manually**: After purchasing additional credits, manually [resume](https://docs.seqera.io/platform-cloud/launch/cache-resume) paused pipelines.

Author: ```suggestion
5. **Resume runs manually**: After purchasing additional credits, manually [resume](../launch/cache-resume) paused pipelines.
```
